src/odemis/gui/cont/streams.py

```diff
@@ -544,6 +548,31 @@ def _on_metadata_btn(self, evt):
                         v = u"[%s … %s]" % (u", ".join(str(a) for a in v[:20]), u", ".join(str(a) for a in v[-20:]))
                     text += u"%s: %s\n" % (key, v)
 
+        # only display extra settings once
+        if model.MD_EXTRA_SETTINGS in raw[0].metadata:
+            text += u"\n======================================\nHardware Settings\n"
+            text += u"======================================\n"
+            for comp, vas in md[model.MD_EXTRA_SETTINGS].items():
+                try:
+                    if vas:
+                        text += u"Component %s:\n" % comp
+                    for name, (value, unit) in vas.items():
+                        unit = unit or ""  # don't display 'None'
+                        unit = unit if value is not None else ""  # don't display unit if data is None (None Hz doesn't make sense)
+                        if isinstance(value, dict):
+                            if value:
+                                text += u"\t%s:\n" % name
+                                for key, val in value.items():
+                                    text += u"\t\t%s: %s %s\n" % (key, val, unit)
+                            else:
+                                # still display the VA, might be interesting (e.g. that no axis was referenced)
+                                text += u"\t%s: {}\n" % name
+                        else:
+                            text += u"\t%s: %s %s\n" % (name, value, unit)
+                except Exception as ex:
+                    logging.warning("Couldn't display metadata for component %s, failed with exception %s" % (comp, ex))
```

Comment:
You can keep the message a little shorter with just "Couldn't display metadata for component %s: %s".